### PR TITLE
✨  Add Quantcast type for amp-consent

### PR DESCRIPTION
**Pull request by nicobatalla**
Adding Quantcast to the list of registered CMPs.

Added:
- Quantcast CMP config
- Example on cmp-vendors page
- Example documentation.

[Quantcast Choice](https://www.quantcast.com/gdpr/consent-management-solution/) is a consent management provider (CMP) solution built on and registered with the IAB Europe Transparency and Consent framework. 

Additional context
We would like to register Quantcast Choice as a Supported Consent Management Platform for visibility and ease of implementation. 
Context: https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/integrating-consent.md

cc @ampproject/wg-monetization

### CODE REVIEW

File: examples/amp-consent/cmp-vendors.amp.html [hunk 1]
```diff
@@ -318,6 +319,24 @@ <h3>Image that is NOT blocked by consent</h3>
...
+              "postPromptUI": "quantcast-post-prompt",
+              "clientConfig": {
+                "coreConfig": {
+                  "googleEnabled": false
+                }
+              }
+            }
+          </script>
+          <div id="quantcast-post-prompt">
+            <button on="tap:quantcast.prompt()" role="button">Privacy settings</button>
```
Comment (micajuine-ho) [inline examples/amp-consent/cmp-vendors.amp.html:335]: I think this has to be `tap:consent.prompt(consent=quantcast)`

File: extensions/amp-consent/cmps/quantcast.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            "postPromptUI": "quantcast-post-prompt",
+            "clientConfig": {
+              "coreConfig": {
+                "googleEnabled": true
+              }
+            }
+          }
+        </script>
+        <div id="quantcast-post-prompt">
+          <button on="tap:quantcast.prompt(consent=quantcast)" role="button">Privacy settings</button>
```
Comment (micajuine-ho) [inline extensions/amp-consent/cmps/quantcast.md:34]: Same thing here: `tap:consent.prompt(consent=quantcast)`

**Review by micajuine-ho — COMMENTED** [review]
Hi @nicobatalla! Thank you for your contribution. I am trying to check the implantation locally, but I'm running into a CORS error:
image

Can you add the `'Access-Control-Allow-Origin'` header to your `checkConsentHref` endpoint?

**Comment by nicobatalla** [comment]
Hi @micajuine-ho. I addressed both your suggestions. Also, the endpoint is working now. Thank you!

**Review by micajuine-ho — APPROVED** [review]
Looks great!

File: extensions/amp-consent/0.1/cmps.js [hunk 3]
```diff
@@ -91,6 +91,12 @@ CMP_CONFIG['opencmp'] = {
   'promptUISrc': 'https://cdn.opencmp.net/tcf-v2/amp/cmp.html',
 };
 
+CMP_CONFIG['quantcast'] = {
+  'consentInstanceId': 'quantcast',
+  'checkConsentHref': 'https://apis.quantcast.mgr.consensu.org/amp/check-consent',
```
Comment (micajuine-ho) [inline extensions/amp-consent/0.1/cmps.js:96]: nit: enter after `'checkConsentHref': ` or run `gulp --lint`

Comment (nicobatalla) [reply extensions/amp-consent/0.1/cmps.js]: done!

**Pull request merged**